static/app/views/insights/pages/mobile/settings.ts

```diff
@@ -2,3 +2,14 @@ import {t} from 'sentry/locale';
 
 export const MOBILE_LANDING_SUB_PATH = 'mobile';
 export const MOBILE_LANDING_TITLE = t('Mobile');
+
+export const OVERVIEW_PAGE_ALLOWED_OPS = [
+  'ui.action.swipe',
+  'ui.action.scroll',
+  'ui.action.click',
+  'ui.load',
+  'app.lifecycle',
+  // navigation and pageload are seen in react-native
+  'navigation',
+  'pageload',
```

Comment:
I wonder if there's anything else we could additionally filter on to get rid of browser navigations and pageloads 🤔 Anyway this PR is an improvement regardless

Reply:
Yeah that would be great! Also, I think ideally we are able to filter by projects directly.

For example in the frontend view, we only show data for frontend projects. That way we don't have to worry about filtering by the right transaction ops, and custom transactions would work too.